pages/article/[id].js

```diff
@@ -67,6 +76,8 @@ function ArticlePage({ query }) {
     [data, dataForUser]
   );
 
+  const replySectionRef = useRef(null);
```

Comment:
what's that ref used for?

Reply:
Ohh good question. It's not used for this PR.
After an editor submits a new reply, the ref is used to scroll the user to the new reply (top section of the reply).

The logic can be seen here in the old codebase: https://github.com/cofacts/rumors-site/blob/master/pages/article.js#L109-L114